lib/posix/cond.c

```diff
@@ -202,4 +202,39 @@ static int pthread_cond_pool_init(void)
 
 	return 0;
 }
+
+int pthread_condattr_init(pthread_condattr_t *att)
+{
+	__ASSERT_NO_MSG(att != NULL);
+
+	att->clock = CLOCK_REALTIME;
```

Comment:
I would recommend this is set to `CLOCK_MONOTONIC` by default.

Reply:
Done, changed default value to clock monotonic